src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -112,6 +117,31 @@ const AccessibilityRequestDetailPage = () => {
     });
   };
 
+  const [removeDocumentMutation] = useMutation<
```

Comment:
I'm curious why this needed to be moved from where it was to this view.  I don't have a problem with it- just don't have context.

Reply:
When we load this component in storybook, it tries to make this graphql call... which fails. Also, it's been our practice to put mutations in top-level View components, so this is doing that